.github/workflows/build.yaml

```
@@ -14,12 +14,20 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        pyversion: [3.6, 3.7, 3.8, 3.9]
         os: [macos-latest, ubuntu-latest, windows-latest]
 
+        golang_version: [1.16.4]
+        java_version: ['15']
```

```suggestion
        java_version: ['16']
```

Any reason not to pick JDK 16?

---

Tests will break 😅
Additionally I'd then like to have tests for have 15 and 16 (sort of smoke test).